<launch>
  <node pkg="first_project" name="gps_to_odom" type="gps_to_odom">
    <param name="lat_r" value="45.477669" />
    <param name="lon_r" value="9.226740" />
    <param name="alt_r" value="169.039000" />
  </node>
</launch>
